### PR TITLE
Fix exponential scheduler with bias

### PR DESCRIPTION
**Pull request by andrey-churkin**
Looks like that current implementation does not work properly.
Consider the following example:
```python
a, b, k = _init_exp(10, 0.1, 0.7)
x = a * np.exp(-k * 10) + b
print(x)
```
I expected that `x` is equal to `0.7` but I got `0.666666666666667`.

### CODE REVIEW

**Comment by andrey-churkin** [comment]
Jenkins please retry a build

File: nncf/common/pruning/schedulers.py [hunk 1]
```diff
@@ -194,11 +202,12 @@ def get_b(a, k):
             return p_min - a
```
Comment (alexsu52) [inline nncf/common/pruning/schedulers.py:202]: def get_b(a, k): -> def get_b(a):

**Review by alexsu52 — CHANGES_REQUESTED** [review]

File: nncf/common/pruning/schedulers.py (hunk 1 above)
Comment (andrey-churkin) [reply nncf/common/pruning/schedulers.py:202]: Done.

**Review by alexsu52 — APPROVED** [review]

**Review by mkaglins — APPROVED** [review]

**Comment by vanyalzr** [comment]
@mkaglins @AlexKoff88 I suppose we could re-run some filter pruning benchmarks to see diff in accuracy for this scheduler once this fix is introduced?

**Comment by mkaglins** [comment]
> @mkaglins @AlexKoff88 I suppose we could re-run some filter pruning benchmarks to see diff in accuracy for this scheduler once this fix is introduced?

This `exponential_with_bias` scheduler is not used in any officially claimed results, but we can run a comparison of `exponential` (that often used in Imagenet experiments) with this fixed version.

File: nncf/common/pruning/schedulers.py [hunk 2]
```diff
@@ -190,17 +198,18 @@ def _init_exp(epoch_idx, p_min, p_max, factor=0.125):
...
         def get_a(k):
-            return (3 / 4 * p_max - p_min) / (np.exp(-k * factor * epoch_idx) - 1)
+            return (p_max - p_min) / (np.exp(-k * epoch_idx) - 1)
 
         def f_to_solve(x):
-            y = np.exp(factor * x * epoch_idx)
-            return 1 / 3 * y + 1 / (y ** 7) - 4 / 3
+            c = (0.75 * p_max - p_min) / (p_max - p_min)
+            y = np.exp(-x * epoch_idx)
+            return y ** factor - c * y + c - 1
```
Comment (vanyalzr) [inline nncf/common/pruning/schedulers.py:210]: Should we make the `factor` parameter adjustable via config? I suppose the 1/8 value is based on what is set in the paper, not on our empirical results.

Comment (mkaglins) [reply nncf/common/pruning/schedulers.py:210]: Yes, it is a good idea to set better default value based on empirical results on our models (firstly get these results). But I am not sure about the importance of this param in the config for users.

**Pull request merged**